control-plane/plugin/src/operations.rs

```
@@ -70,6 +74,13 @@ pub trait Scale {
     async fn scale(id: &Self::ID, replica_count: u8, output: &utils::OutputFormat) -> PluginResult;
 }
 
+/// Set trait.
+/// To be implemented by resources which support the 'set' operation.
+#[async_trait(?Send)]
+pub trait Set {
+    type ID;
+    async fn set(id: &Self::ID, prop_name: &str, prop_value: &str, output: &utils::OutputFormat);
```

This should return an error..